packages/fhir-converter/src/templates/cda/DataType/Coding.hbs

```diff
@@ -25,7 +25,7 @@
   // -------------------------------------------------------------------------------------------------
 --}}
 {
-	"code":"{{code.code}}",
+	"code": {{#if code.code}} "{{trim code.code}}" {{else if canBeUnknown}} "UNK" {{/if}},
```

Comment:
based on my previous comment - can't we fill out the `code`, `system`, and `display` in the `UNK` case?